pyproject.toml

```diff
@@ -29,7 +29,7 @@ dependencies = [
     "flask-talisman==1.1.0",
     "flask-wtf==1.2.2",
     "flask==3.0.3",
-    "govuk-frontend-jinja==2.7.0",
+    "govuk-frontend-jinja==3.4.0",
```

Comment:
I think this is problematic - each specific release of govuk-frontend-jinja is designed to work with a specific GOV.UK Frontend. It feels like this would probably break the existing apply/assess stuff and we should do some heavy manual QA/checks. Unless you've already done that?

Reply:
Rolled back